standalone/index.html

```diff
@@ -209,7 +246,7 @@
     </style>
   </head>
   <body>
-    <h1>WebGPU Conformance Test Suite</h1>
+    <h1><img class="logo" src="webgpu-logo-notext.svg"></img>WebGPU Conformance Test Suite</h1>
```

Comment:
```suggestion
    <h1><img class="logo" src="webgpu-logo-notext.svg">WebGPU Conformance Test Suite</h1>
```
Closing tag only needed in XHTML, but we're HTML5 :)